### PR TITLE
[ Website ]: Progressive enhancement

### PR DESCRIPTION
**Pull request by joshbruce**

### CODE REVIEW

**Comment by joshbruce** [comment]
See also: #1066 

Adding conversationalists: @carodew, @kategarklavs, @rogeruiz, @juliaelman, @maya. 

Went a complete less verbose route with this one. Let me know what y'all think.

**Comment by joshbruce** [comment]
@kategarklavs - I'm also horrible at the whole affecting and effecting thing...sorry if I chose poorly (now I want to watch Indiana Jones).

**Comment by rogeruiz** [comment]
Thanks @joshbruce. The less verbose route seems to be easier to understand. Because this is much different than the changes in #1066, @kategarklavs could you look through this version to see if there is anything that should be edited? Thanks! 👍

**Comment by kategarklavs** [comment]
@joshbruce No apologies necessary! I'm glad to take another look. And now I also want to watch "Indiana Jones" 😁

File: docs/pages/getting-started.html [hunk 1]
```diff
@@ -52,6 +52,12 @@ <h3 class="usa-heading">For developers</h3>
 
 <p>The standards are designed to support both older and newer browsers through progressive enhancement, and officially support Internet Explorer 9 and up, and the latest versions of Chrome, Firefox, and Safari. Internet Explorer 8 and below generally see very low usage today, and most agency websites should be able to safely begin support at Internet Explorer 9.</p>
 
+<h4>Progressive enhancement:</h4>
+
+<p>Progressive enhancement is a web design and development strategy emphasizing accessibility through <a href="http://www.w3.org/html/wg/wiki/Guide/Syntax-Semantics" target="_blank" title="World Wide Web Consortium - Guide/Syntax and Semantics">semantic HTML markup</a>. Semantic HTML allows older browsers to present the content of a document to users. The semantic markup is layered with external technologies to enhance the user experience (specifically, CSS and JavaScript), without necessarily effecting the underlying funcitonality of the document and site.</p>
```
Comment (kategarklavs) [inline docs/pages/getting-started.html]: Hey! A few notes here:

I'd reword this:

layered with external technologies to enhance the user experience (specifically, CSS and JavaScript)

to this:

layered with external technologies (specifically, CSS and JavaScript) to enhance the user experience 

(Does that make sense/is that still correct?)

Also, please change `effecting` to `affecting`

Comment (kategarklavs) [inline docs/pages/getting-started.html]: Also! Noticed that `functionality` is misspelled — please update!

**Comment by maya** [comment]
Hi @joshbruce, thanks for making a new PR. We currently have content about progressive enhancement here: https://github.com/18F/web-design-standards/pull/1184/files#diff-c5ba760212cf8eed5d2784b94da8b680R12 and having it in two places is not ideal.

Before we’ll consider publishing this, our team needs to make sure the content aligns with observed user needs. We’ll also want to do a full edit (not in GitHub) to make sure the content mirrors the voice and tone of our existing work.

**Comment by maya** [comment]
Hi @joshbruce, thanks for making a new PR. We currently have content about progressive enhancement here: https://github.com/18F/web-design-standards/pull/1184/files#diff-c5ba760212cf8eed5d2784b94da8b680R12 and having it in two places is not ideal.

Before we’ll consider publishing this, our team needs to make sure the content aligns with observed user needs. We’ll also want to do a full edit (not in GitHub) to make sure the content mirrors the voice and tone of our existing work.

File: docs/pages/getting-started.html (hunk 1 above)
Comment (joshbruce) [inline docs/pages/getting-started.html]: Commenting via email reply.

Getting booted every time I try to sign in.

Also, every time I try to update the PR via branch commit - the PR is not updated.(https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request <https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request>). This was the same workflow used for the aria-hidden PR we did for #1160 (https://github.com/18F/web-design-standards/pull/1160). So, I’m wondering if, for some reason, the inline notes are causing an issue somehow. Or, if it is strictly a problem with sessioning - and the service problems they are experiencing: https://status.github.com

Anyway. Have the changes made. They are queued. Now, we wait, I suppose.

Comment (maya) [inline docs/pages/getting-started.html]: Josh, I'm getting the same thing. My comment keeps disappearing and
reappearing. Looks like github is having some issues today!

On Mon, May 23, 2016 at 9:02 AM, Josh Bruce <notifications@github.com>
wrote:

> In docs/pages/getting-started.html
> <https://github.com/18F/web-design-standards/pull/1184#discussion_r64244730>
> :
>
> > @@ -52,6 +52,12 @@ <h3 class="usa-heading">For developers</h3>
> >
> >  <p>The standards are designed to support both older and newer browsers through progressive enhancement, and officially support Internet Explorer 9 and up, and the latest versions of Chrome, Firefox, and Safari. Internet Explorer 8 and below generally see very low usage today, and most agency websites should be able to safely begin support at Internet Explorer 9.</p>
> >
> > +<h4>Progressive enhancement:</h4>
> > +
> > +<p>Progressive enhancement is a web design and development strategy emphasizing accessibility through <a href="http://www.w3.org/html/wg/wiki/Guide/Syntax-Semantics" target="_blank" title="World Wide Web Consortium - Guide/Syntax and Semantics">semantic HTML markup</a>. Semantic HTML allows older browsers to present the content of a document to users. The semantic markup is layered with external technologies to enhance the user experience (specifically, CSS and JavaScript), without necessarily effecting the underlying funcitonality of the document and site.</p>
>
> Commenting via email reply. Getting booted every time I try to sign in.
> Also, every time I try to update the PR via branch commit - the PR is not
> updated.(
> https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request <
> https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request>).
> This was the same workflow used for the aria-hidden PR we did for #1160
> <https://github.com/18F/web-design-standards/pull/1160> (#1160
> <https://github.com/18F/web-design-standards/pull/1160>). So, I’m
> wondering if, for some reason, the inline notes are causing an issue
> somehow. Or, if it is strictly a problem with sessioning - and the service
> problems they are experiencing: https://status.github.com Anyway. Have
> the changes made. They are queued. Now, we wait, I suppose.
> … <#m_-1873645326641936713_>
> On May 23, 2016, at 10:54 AM, Kate Garklavs ***@***.***> wrote: In
> docs/pages/getting-started.html <#1184 (comment)
> <https://github.com/18F/web-design-standards/pull/1184#discussion_r64236503>>:
> > @@ -52,6 +52,12 @@ <h3 class="usa-heading">For developers</h3> > > <p>The
> standards are designed to support both older and newer browsers through
> progressive enhancement, and officially support Internet Explorer 9 and up,
> and the latest versions of Chrome, Firefox, and Safari. Internet Explorer 8
> and below generally see very low usage today, and most agency websites
> should be able to safely begin support at Internet Explorer 9.</p> > >
> +<h4>Progressive enhancement:</h4> > + > +<p>Progressive enhancement is a
> web design and development strategy emphasizing accessibility through <a
> href="http://www.w3.org/html/wg/wiki/Guide/Syntax-Semantics"
> target="_blank" title="World Wide Web Consortium - Guide/Syntax and
> Semantics">semantic HTML markup</a>. Semantic HTML allows older browsers to
> present the content of a document to users. The semantic markup is layered
> with external technologies to enhance the user experience (specifically,
> CSS and JavaScript), without necessarily effecting the underlying
> funcitonality of the document and site.</p> Also! Noticed that
> functionality is misspelled — please update! — You are receiving this
> because you were mentioned. Reply to this email directly or view it on
> GitHub <
> https://github.com/18F/web-design-standards/pull/1184/files/a754fd75d31c84c48e82a6d67823100392c7968e#r64236503
> >
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/web-design-standards/pull/1184/files/a754fd75d31c84c48e82a6d67823100392c7968e#r64244730>
>



-- 
Maya Benari
Innovation Specialist, 18F <http://18f.gsa.gov>
maya.ben-ari@gsa.gov

**Comment by joshbruce** [comment]
@maya - No worries - do what needs doing to suit the mission - just trying to give back where I can. ;)

If I may speak as a user of the Standards, to put two cents in (so to speak): 

For the project I am on - the idea of progressive enhancement is pretty foreign to the developers we have. So, when I saw the “progressively enhanced” mention - I was elated, because I know what it means. When I mentioned it to our developers who were not doing things in a progressive enhancement manner, there was (read “is") a lot of confusion and pushback. 

Which prompted me to update the standards I’m responsible for (which are an extension of the USWDS) and submit the initial PR to see about promoting the idea and clarification to the broader user community.

And I understand the desire to explore user need - I’m one of the resident UX people on my project - and for that project, it was definitely necessary. :)

Just throwing copper.

**Comment by joshbruce** [comment]
I'm going to step away from this until GitHub gets back up and running a little bit more. Seems CircleCI is not running. Will most likely check back in tomorrow.

**Comment by kategarklavs** [comment]
@joshbruce Hello! Thank you again for your insight and contributions — we very much appreciate them. For now, we're going to close this PR, but we'll be sure to revisit it after we conduct more research around this topic. Meanwhile, let us know if you have any questions.
